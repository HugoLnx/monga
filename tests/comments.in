/* test */
"before1"/* "test" */"after1"
"before2" /* "test */ "after2"
"before3" /*"test*/ "after3"
/**//***//****//* */
"before4"/*"*/"after4"
/** abc **/ /** abc */ /* abc **/ /* a*bc */ /* a**bc */ /* line1
line2*/
