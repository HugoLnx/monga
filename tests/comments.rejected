/* abcd
/** abc
/** abc"
/** "abc
/*/
/*
abc */
abc **/
"abc **/
abc" **/
