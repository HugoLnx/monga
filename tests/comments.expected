COMMENT,/* test */
TEXT,"before1"
COMMENT,/* "test" */
TEXT,"after1"
TEXT,"before2"
COMMENT,/* "test */
TEXT,"after2"
TEXT,"before3"
COMMENT,/*"test*/
TEXT,"after3"
COMMENT,/**/
COMMENT,/***/
COMMENT,/****/
COMMENT,/* */
TEXT,"before4"
COMMENT,/*"*/
TEXT,"after4"
COMMENT,/** abc **/
COMMENT,/** abc */
COMMENT,/* abc **/
COMMENT,/* a*bc */
COMMENT,/* a**bc */
COMMENT,/* line1
line2*/
